<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorListWidget</class>
 <widget class="QWidget" name="SensorListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Czujniki pomiarowe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dostepne czujniki na stacji</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="stationNameLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nazwa stacji</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="sensorListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="sensorCountLabel">
     <property name="text">
      <string>Dostepne czujniki: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Powrot do listy stacji</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>